<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AutoScope</class>
 <widget class="QMainWindow" name="AutoScope">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AutoScope</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow{

	
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTreeView" name="treeView">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>141</width>
      <height>551</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="videoLabel">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>100</y>
      <width>491</width>
      <height>341</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
	background-color:black
}</string>
    </property>
    <property name="text">
     <string>视频</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="fileMenu">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="loadProjectAction"/>
   </widget>
   <widget class="QMenu" name="communicationMenu">
    <property name="title">
     <string>通信</string>
    </property>
    <addaction name="action"/>
    <addaction name="action_2"/>
   </widget>
   <widget class="QMenu" name="settingMenu">
    <property name="title">
     <string>设置</string>
    </property>
    <addaction name="connectSettinggAct"/>
    <addaction name="pathSettingAct"/>
   </widget>
   <widget class="QMenu" name="screenCaptrue">
    <property name="title">
     <string>截屏</string>
    </property>
   </widget>
   <widget class="QMenu" name="screenRecording">
    <property name="title">
     <string>录屏</string>
    </property>
   </widget>
   <addaction name="fileMenu"/>
   <addaction name="communicationMenu"/>
   <addaction name="settingMenu"/>
   <addaction name="screenCaptrue"/>
   <addaction name="screenRecording"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="loadProjectAction">
   <property name="text">
    <string>加载项目</string>
   </property>
  </action>
  <action name="action">
   <property name="text">
    <string>连接</string>
   </property>
  </action>
  <action name="action_2">
   <property name="text">
    <string>断开连接</string>
   </property>
  </action>
  <action name="connectSettinggAct">
   <property name="text">
    <string>连接设置</string>
   </property>
  </action>
  <action name="pathSettingAct">
   <property name="text">
    <string>路径设置</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
